pallets/invulnerables/Cargo.toml

```diff
@@ -12,15 +12,20 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 log = { workspace = true }
 parity-scale-codec = { workspace = true }
+rand = { version = "0.8.5", features = ["std_rng"], default-features = false }
```

Comment:
rand should be marked as optional since AFAIK is only used in benchmarks. It should be only incorporated with runtime-benchmarks

Reply:
Same for pallet-session and pallet-balances

Reply:
And it should use workspace = true:

```
rand = { workspace = true, features = ["std_rng"], optional = true }
```

Reply:
done except `pallet_session` which is used in the pallet to implement `SessionManager`.